### ISSUE TITLE
umb-multiple-textbox does not have an accessible label on input elements

### ISSUE THREAD

**Issue by lordscarlet** [umbraco/Umbraco-CMS#8725]
The input fields created by _Umbraco.Multipletextstring_ do not have an accessible label

## Umbraco version

I am seeing this issue on Umbraco version: 8.6.1

Reproduction
------------

The property editor for _Multiple Text String_ makes no attempt to provide an accessible label. 

### Specifics

https://github.com/umbraco/Umbraco-CMS/blob/44b84c4cddb44a60b6ea73fe5870c251328fa857/src/Umbraco.Web.UI.Client/src/views/propertyeditors/multipletextbox/multipletextbox.html#L5-L9

### Steps to reproduce

1. Create a document type containing a property using the _Umbraco.Multipletexstring_ editor
2. Create a node using the created content type
3. Add a test string to reveal an input

### Expected result

Each input field should have an associated `<label>` with a `for` attribute referencing the `id` of the `<input>` field

### Actual result

The `<input>` has no label

**Comment by umbrabot** [comment]
Hi @lordscarlet,

We're writing to let you know that we would love some help with this issue. We feel that this issue is ideal to flag for a community member to work on it. Once flagged here, folk looking for issues to work on will know to look at yours. Of course, please feel free work on this yourself ;-).  If there are any changes to this status, we'll be sure to let you know. 

For more information about issues and states, have a look at [this blog post](https://umbraco.com/blog/the-umbraco-issue-tracker-process/)

Thanks muchly, from your friendly Umbraco GitHub bot :-)

**Comment by lordscarlet** [comment]
@nul800sebastiaan at risk of being confusing, I am going to do somewhat of a cross-post here. On #8726 we are discussing a similar situation, and I mentioned two options. From a UI perspective, option 1 (`aria-labelledby`) would look like this:

![image](https://user-images.githubusercontent.com/17951/91844117-e00abf80-ec24-11ea-962f-a1b248df8147.png)

It passes [ANDI](https://github.com/SSAgov/ANDI) validation (which matters for my implementation, others may have different standards)

The alternative would look like this (possibly with some different formatting in the end? Maybe it's fine):

![image](https://user-images.githubusercontent.com/17951/91849138-91135900-ec29-11ea-86ef-ffec10dedb41.png)

I don't know what the preference is from a design/UX side in balance with accessibility.

**Comment by lordscarlet** [comment]
I am going to go ahead and submit a PR for the second option, as I think it is the most accessibility-friendly version. I'm still not sure on my actual implementation because it uses multiple nested `$parent` references, but I thought I would get it out there for review just in case.

**Comment by tiffy74** [comment]
I think aria-labelled by along with aria-describedby would be preferable since the item is effectively a single component and labelledby is a better option for giving the user an understanding that each field is related to a single function. The risk of using pure labels is that you would lose the context/association of the parent component.  However labelledby adds it's own challenges as it loses some of the useful bits that labels provide such as the ability to click on the label to give focus on the input.
So I would say labelledby is better for this kind of component.. but with caution.
As with all accessibility, the only way to truly tell is if it is tested by a range of users with differing accessibility needs.

**Comment by umbrabot** [comment]
Hiya @lordscarlet!

We wanted to thank you for your work and let you know that we have spent some time reviewing the contribution and have decided that we are going to close this one. 

While reviewing this PR, the team left the following additional note:  
 > Closing this in relation to #8790 - make sure to read the close reason on #8790

We appreciate that you have worked hard to build and document this contribution to create the changes you want to see but given the status of Umbraco 8, [moving into LTS](https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/) and Umbraco 9 being ready and stable and packed with new features, we are working through the open PRs and deciding which are a good fit for the last version 8 minor and which are not. 

If you'd like to know a little more about this process, [please check out this blog post](https://umbraco.com/blog/shaping-up-the-umbraco-818-release/), explaining how we came to the decisions we have made here. We'd like to reassure you though that the CMS team, along with some special guests, took the time to assess each PR that is being closed and decided how to proceed - whilst the message you're receiving is automatic, the work behind it was done by the people that make up the CMS team, along with community members. 

If there are specifics around this PR that you'd like explained that are not covered by the article, please let us know.

All the best to you, our wonderful contributor.

Thanks, from your friendly Umbraco GitHub bot 🤖 🙂
